content/post/NEW-ipfs-camp-recap.md

```diff
@@ -0,0 +1,54 @@
+- [**⚡️ Lightning Talks**](https://github.com/ipfs/camp#%EF%B8%8F-lightning-talks) - Learn what is the latest from multiple projects in the IPFS ecosystem
+- [**🧬 Sci-Fi Fair**](https://github.com/ipfs/camp#-sci-fi-fair) - Get blown away by huge Sci-Fi experiments, soon to be seen in a Network close to you
+- [**🧩 Unconf**](https://github.com/ipfs/camp#-unconf) - Gather folks around shared topics of interest and discuss collaborations.
+
+### IPFS Camp `Succes`s is `defined` by the number of `friendships`, `conversations`, `collaborations` and `new projects` that get started.
+
+And it was a blast. Here is a quick video recap as a souvenir for everyone that was there and a postcard for everyone that didn't get a chance this time:
+
+INSERT RECAP VIDEO HERE
+
+However, the IPFS Camp spirit doesn't stop here, now it is all about continuing the dialog that got started and taking all the learnings and materials to local IPFS Communities, showing and teaching more folks how to build on the P2P Web. All the content produced can be found on the IPFS Camp Repo -- http://github.com/ipfs/camp. Soon we will be posting all the recordings as well.
+
+Last, but definitely not the least, a huge THANK YOU ❤️ to everyone that contributed to the event. IPFS Camp wouldn't be possible if it werent for:
+
+- **160 Attendees** coming from over **22 Countries**!
+- [**67 Content Creators**](https://camp.ipfs.io/team) that produced all the training and talk material for the camp attendees
+- An outstanding [Production Team](https://camp.ipfs.io/team) that created the phenomenal experienced for everyone.
+- The over 4000 IPFS Community members that have been building this project with us since the beginning.
+
+INSERT GROUP PHOTO HERE
+
+In name of the whole IPFS Project, thank you all for being part of this project, such an ambitious goal couldn't be achieved without all your energy, passion and will to do the right thing.
+
+David Dias & IPFS Team
+
+`Extra:` As a closing gift, here are some random interesting facts about Camp:
+
+- IPFS Camp was the 1st time the Lazer Cat bot became **P2P Lazer Cat bot** 😸
+- All you can eat fruit ice cream made all attendees very happy 🍨
+- Textile created a **P2P Tag Game**. I believe the last person to be "it" is still looking the next person to tag, so look out at next events! 👋🏽
+- The first **SWAG Trading Post** was established, an eco-responsible way of attendees picking the item of swag they love the most or that they can trade for others, avoiding the regular Conference waste of ETOOMANYBELLSANDWHISTLES.
+- The Wifi worked! Well, almost, we are sorry for that one morning hiccup. Thank you everyone for the patience and the Trainers for coming prepared with ways to make their courses work offline and distributed ❤️
+- The whole NPM was crammed into a box for the first time, an homage to Jen from the IT Crowd
```

Comment:
@achingbrain what is the best photo for me to link here?

Reply:
my fave: ![image](https://user-images.githubusercontent.com/618519/60735173-99530400-9f52-11e9-9b88-555702d633c9.png)